doc/workshop/ARMA/inputs/plot_synth.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+
+import sys
+import os
+import matplotlib.pyplot as plt
+import pandas as pd
+
+def main():
+    try:
+        synth_fp = os.path.abspath(sys.argv[1])
+        real_fp = os.path.abspath(os.path.join(os.path.dirname(synth_fp), "dataSet_0.csv"))
```

Comment:
If possible, I suggest to use camelBack for the variables.

Reply:
Fixed